src/instance.rs

```diff
@@ -18,14 +14,12 @@ impl Instance {
         unsafe {
             rtcSetGeometryInstancedScene(h, scene.handle);
         }
-        Arc::new(Instance {
-            handle: h,
-            scene: scene,
-        })
+        Arc::new(Instance { handle: h, scene })
```

Comment:
Do you also plan to or should we remove the Arc from Instance as well? I think it can get the same treatment to use rtcRetain/rtcRelease as the other objects (I may just be reviewing this before you've gotten to it, if so no problem :) )

Reply:
I removed the Arc from instance, scene now managed internall by embree.